download.tt

```diff
@@ -0,0 +1,339 @@
+</p>
+
+<ul id="getting-nix" class="nav nav-tabs">
+  <li class="active"><a href="#nix-quick-install">Quick install</a></li>
+  <li><a href="#nix-verify-installation">Verify installation (GPG)</a></li>
+  <li><a href="#nix-uninstall">Uninstall</a></li>
+  <li><a href="#nix-more">More ...</a></li>
+</ul>
+
+<div class="tab-content">
+  <section class="tab-pane active" id="nix-quick-install">
+    The quickest way to install Nix is to run the following in a shell (as a
+    user other than <code>root</code>):
+    <pre class="well" style="margin-top:1em">
+<span class="nix-command">curl -L --proto '=https' --tlsv1.2 https://nixos.org/nix/install | sh</span></pre>
+    <p>Make sure to follow the instructions output by the script.</p>
+    <p>The installation script requires that you have <code>sudo</code> access to <code>root</code>.</p>
+  </section>
+  <section class="tab-pane" id="nix-verify-installation">
+    <p>You may want to verify the integrity of the installation script using GPG:</p>
+    <pre class="well">
+<span class="nix-command">curl -o install-nix-[%latestNixVersion%] https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install</span>
+<span class="nix-command">curl -o install-nix-[%latestNixVersion%].asc https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install.asc</span>
+<span class="nix-command">gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE</span>
+<span class="nix-command">gpg2 --verify ./install-nix-[%latestNixVersion%].asc</span>
+<span class="nix-command">sh ./install-nix-[%latestNixVersion%]</span></pre>
+    <p>
+      The <a href="[%root%]edolstra.gpg">signing key</a> has fingerprint
+      <code>B541 D553 0127 0E0B CF15 CA5D 8170 B472 6D71 98DE</code>.
+      It is also available
+      <a href="https://github.com/NixOS/nixos-homepage/blob/master/edolstra.gpg">on GitHub</a>.
+    </p>
+  </section>
+  <section class="tab-pane" id="nix-uninstall">
+    <p>
+      You can uninstall Nix simply by running <code>rm -rf /nix</code>.
+    </p>
+    <!-- TODO: expand this https://github.com/NixOS/nix/issues/1402 -->
+  </section>
+  <section class="tab-pane" id="nix-more">
+    <p>See <a href="[%nixManual%]#chap-installation">the manual</a> for more information.</p>
+    <p>The following artifacts are also available:</p>
+    <ul>
+      <li><a href="[%latestNixMirror%]/nix-[%latestNixVersion%].tar.xz">Source tarball</a>.</li>
+      <li><a href="[%nixManual%]#ssec-relnotes-[%latestNixVersion%]">Release notes</a>.</li>
+      <li>
+        <a href="[%nixManual%]">Manual</a>.
+        Please read the <a href="[%nixManual%]#chap-quick-start">“Quick Start” section of the manual</a>
+        for an overview of how to install and use Nix.
+      </li>
+      <li>Old releases are <a href="/releases/nix">also available</a>.</li>
+    </ul>
+  </section>
+</div>
+
+<script charset="utf-8">
+  $('#getting-nix a').click(function (e) {
+    e.preventDefault();
+    $(this).tab('show');
+  })
+
+</script>
+
+
```

Comment:
```suggestion
<br><br>

```

add some space (not tested, see if it looks better or even more is needed)

![Screenshot from 2020-04-11 09-59-57](https://user-images.githubusercontent.com/91113/79038660-53ab2200-7bdb-11ea-97ec-c9cfbe952587.png)